webapp/channels/src/components/root/root.tsx

```diff
@@ -549,75 +549,77 @@ export default class Root extends React.PureComponent<Props, State> {
                         <GlobalHeader/>
                         <CloudEffects/>
                         <TeamSidebar/>
-                        <Switch>
-                            {this.props.products?.filter((product) => Boolean(product.publicComponent)).map((product) => (
-                                <Route
-                                    key={`${product.id}-public`}
-                                    path={`${product.baseURL}/public`}
-                                    render={(props) => {
-                                        return (
+                        <div className='main-wrapper'>
```

Comment:
This should be fine, but I want to call out that we should test thoroughly (incl. running all E2E) since adding another div at the top level of the app can affect a lot of things.

If there's a way we can avoid adding this, we probably should.

Reply:
@devinbinnie I hoped to avoid this as well, but this was the only way I could properly group the LHS, Center Panel and RHS for this UI layout tweak